### ISSUE TITLE
Allow resources or methods to be tagged with multiple swagger tags

### ISSUE THREAD

**Issue by enrique-guillen** [Apipie/apipie-rails#632]
The [`swagger-editor`](https://swagger.io/docs/open-source-tools/swagger-editor) allows clicking on tags and getting all endpoints tagged with that tag. [It seems like](https://github.com/Apipie/apipie-rails/blob/61ad1e69fc98ee248f3f1181830a40d53aae7d41/lib/apipie/swagger_generator.rb#L227) Apipie only supports one customizable tag, which I'm guessing is the name of the class or the `name` in the `resource_description`.

The DSL could work like this:

```ruby
  resource_description do
    name 'Foo | Bar'
    tags 'Foo', 'Bar', 'Baz'
    short 'A bar in the foo namespace that is related to Bazes'
  end
```

And/or:

```ruby
  api :POST, 'foo/bars'
  param :relationships, desc: 'A qux object related to the bar can be created here.'
  resource_tags 'Foo', 'Bar', 'Baz', 'Qux'
  def index; ...; end
```

**Comment by enrique-guillen** [comment]
PR no longer a WIP.

**Comment by iNecas** [comment]
Closed by #634, thanks @enrique-guillen
